packages/client/src/v2-events/features/events/registered-fields/Paragraph.tsx

```diff
@@ -0,0 +1,19 @@
+/*
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at https://mozilla.org/MPL/2.0/.
+ *
+ * OpenCRVS is also distributed under the terms of the Civil Registration
+ * & Healthcare Disclaimer located at http://opencrvs.org/license.
+ *
+ * Copyright (C) The OpenCRVS Authors located at https://github.com/opencrvs/opencrvs-core/blob/master/AUTHORS.
+ */
+import { FieldProps } from '@opencrvs/commons'
+import React from 'react'
+import { useIntl } from 'react-intl'
+
+export const Paragraph = ({ id, label }: FieldProps<'PARAGRAPH'>) => {
+  const intl = useIntl()
+
+  return <p id={id}>{intl.formatMessage(label)}</p>
```

Comment:
Could we add a comment regarding the id assignment and explain why we need it

Reply:
Good point, probably related to tests why I ported it. I can start without it!

Reply:
It comes from the FieldProps as it's necessary for React's `key={id}` to track renders, and also to map the `{ [id]: value }` -data to backend. It will likely become necessary for tests; and that's likely why our component library's `InputField` and `DateField` deem it necessary at the moment.

I'll remove it from this paragraph example